src/routes/(default)/about/employees.svelte

```diff
@@ -74,6 +75,25 @@
 	]}
 />
 
+<Employee
+	givenName="Sandra Viktoria"
+	familyName="Toftemo"
+	position="Programvareutvikler"
+	phone="+47 406 30 694"
+	email="sandra.toftemo@bjerk.io"
+	image={sandra}
+	socialMedia={[
+		{
+			type: 'linkedin',
+			url: 'https://www.linkedin.com/in/sandra-viktoria-doseth-toftemo-b9b946227/'
```

Comment:
Tip: There's a [way to set a personal URL on LinkedIn](https://www.linkedin.com/help/linkedin/answer/a542685/manage-your-public-profile-url?lang=en)